src/dbos-executor.ts

```diff
@@ -676,6 +677,7 @@ export class DBOSExecutor implements DBOSExecutorContext {
     ) {
       // TODO: Make this transactional (and with the queue step below)
       args = await this.systemDatabase.initWorkflowStatus(internalStatus, args);
+      await debugTriggerPoint(DEBUG_TRIGGER_WORKFLOW_ENQUEUE);
```

Comment:
I wonder if this will add overhead to non-debug production workloads? If so, can we disable this in prod?

Reply:
Would be extremely surprised if the overhead is measurable.

Reply:
In this case, where it is against several database updates, I agree.  However, disabling in prod is not a bad idea.

Reply:
We could get it down to a boolean check pretty easily.  But we can do it later.